README.md

```diff
@@ -33,6 +33,8 @@ Inside a fresh virtualenv, `cd` into the root folder of this repository (`xblock
 ```bash
 $ pip install -U pip wheel
 $ pip install -r requirements-test.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/base.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/test.txt
```

Comment:
Oh, maybe I'm using a different version of virtual environment. Do you have `$VENV` set? If you do, then it's fine to just keep it as it is.